src/steps/Microsoft.DotNet.UpgradeAssistant.Steps.Packages/Analyzers/TargetCompatibilityReferenceAnalyzer.cs

```diff
@@ -26,28 +26,33 @@ public TargetCompatibilityReferenceAnalyzer(IPackageLoader packageLoader, ITarge
 
         public async Task<PackageAnalysisState> AnalyzeAsync(IProject project, PackageAnalysisState state, CancellationToken token)
         {
+            if (project is null)
```

Comment:
This comes up so often, that Taylor added an extension method to make it unnecessary. If you use `project.Required()` the first time you use the project, the `.Required` will do this for you.

Reply:
Changed